### DISCUSSION TITLE
Is there a web interface?

### DISCUSSION THREAD

**Discussion by alevikpes** [UtrechtUniversity/yoda#481] [Q&A] [answered]
Hello!

I saw a nice web interface but can't find mentioning it in the docs. Do you have your own or was it a third party?

**Comment by stsnel** [comment]
Good day,

Yes, the main web interface of Yoda is the Yoda Portal. It is a custom Flask application that has been developed by the Yoda team.

You can log in on it by navigating to https://portal.yoda.test if you use the development VM (after running the main `playbook.yml` playbook to deploy the application and the `test.yml` playbook to add the test data/accounts). You can find the credentials of the development VM in the test role defaults file: https://github.com/UtrechtUniversity/yoda/blob/e94a994098f89467f83e4c1972fa1cb5959f73bd/roles/yoda_test/defaults/main.yml#L9-L23

There is some user documentation available at the UU website, e.g. https://www.uu.nl/en/research/yoda/guide-to-yoda/collaborating-in-yoda . At the moment this information is geared towards UU employees and is still in the process of being updated.

**Reply by alevikpes** [reply]
I am getting just a pretty json page if I go to http://portal.yoda.test:9200/. Am I missing something?

![image](https://github.com/user-attachments/assets/fbbb5219-5039-4012-a792-7153c78cc5d1)

**Reply by stsnel** [reply, marked as answer]
Can you try navigating to https://portal.yoda.test/ on your host system?

**Reply by alevikpes** [reply]
yes, I can see the web interface

**Reply by alevikpes** [reply]
Oh, is it the one from my VM? I thought it was yours 😆

**Reply by alevikpes** [reply]
Maybe a suggestion: show the url in the `vagrant up` output. Something like `Your portal is now available at ...`

**Reply by stsnel** [reply]
Thanks for your feedback, that sound like a great idea to me :+1: 

We'll add a message like that when the playbook finishes.
